pandas/tests/io/formats/test_format.py

```diff
@@ -2846,6 +2846,13 @@ def test_to_string_multindex_header(self):
         exp = "    r1 r2\na b      \n0 1  2  3"
         assert res == exp
 
+    def test_to_string_empty_col(self):
+        # GH 13653 - replace empty strings with \n
+        s = pd.Series(["", "Hello", "World", "", "", "Mooooo", "", ""])
+        res = s.to_string(index=False)
+        exp = "      \n Hello\n World\n      \n      \nMooooo\n      \n      "
+        assert res == exp
```

Comment:
could use `re.match` here

Reply:
Could you elaborate on what exactly you want me to use `re.match` for?

Reply:
you don't have to but i was thinking of `re.match(exp, res)`